README.md

```diff
@@ -53,6 +53,8 @@ npm run start / npm run dev || yarn start / yarn run dev
 ```
 7. Go to [http://localhost:3000](http://localhost:3000/) in your browser.
 
+Note: The application should automatically add the missing campaign "Reward Promotion" by using script called "missing-campaign.js", so there is no need to add it manually.
```

Comment:
`node missing-campaign`?  Also, it would be great to add a verb into the script name as we do some action, so maybe `node add-missing-campaign`?